Assets/Scripts/Player/Interaction/Grabable.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+using NaughtyAttributes;
+using UnityEngine;
+using UnityEngine.Events;
+
+namespace Player.Interaction
+{
+    [RequireComponent(typeof(Rigidbody))]
+    public sealed class Grabable : MonoBehaviour
+    {
+        [SerializeField] private UnityEvent onGrab = new();
+        [SerializeField] private UnityEvent onRelease = new();
+        
+        [field: ShowNonSerializedField] public bool IsOwned { get; private set; }
+    
+        private bool updatePosition;
+        private new Rigidbody rigidbody;
+        private new Collider collider;
```

Comment:
```suggestion
        private Rigidbody rigidbody;
        private Collider collider;
```
Waarom `new` keyword er bij? Je pakt ze toch all in `Awake`.